tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -270,32 +279,53 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
     csinfo_.avgpoolgrad       = "AvgPoolGrad";
     csinfo_.conv2dgradinput   = "Conv2DBackpropInput";
     csinfo_.conv2dgradfilter  = "Conv2DBackpropFilter";
+    csinfo_.lrn               = "LRN";
+    csinfo_.lrngrad           = "LRNGrad";
+    csinfo_.fused_batch_norm = "FusedBatchNorm";
+    csinfo_.fused_batch_norm_grad = "FusedBatchNormGrad";
 
     rinfo_.push_back({csinfo_.conv2d,   csinfo_.mklconv2d,
```

Comment:
Yes it is missing GetMklOpName.

It is not always the case that new name has any relationship with oldname. BiasAddGrad is an example. We are planning to add that op soon.